Sources/SwiftSyntax/SyntaxArenaAllocatedBuffer.swift

```diff
@@ -107,4 +107,8 @@ public struct SyntaxArenaAllocatedBufferPointer<Element: Sendable>: RandomAccess
   var unsafeRawBufferPointer: UnsafeRawBufferPointer {
     return UnsafeRawBufferPointer(buffer)
   }
+
+  public func withContiguousStorageIfAvailable<R>(_ body: (UnsafeBufferPointer<Element>) throws -> R) rethrows -> R? {
+    try body(buffer)
+  }
```

Comment:
I this used anywhere?

Reply:
Yes, `UnsafeMutableBufferPointer.initialize(fromContentsOf:)` have a fast path calling this witness.

Reply:
Ah, I didn’t think about this satisfying a defaulted requirement in `RandomAccessCollection`.